File: python_programming/practices/fizzbuzz.py
# KR 6th FizzBuzz

number = 1 

while number < 51:
    if number % 3 == 0 and number % 5 == 0:
        print(f"fizz buzz")
    elif number % 5 == 0:
        print(f"buzz")
    elif number % 3 == 0:
        print(f"fizz")
    else:
        print(number)
    number += 1

File: c_programming/practices/fizzbuzz.c
//KR 6th fizzbuzz
#include <stdio.h>

int main(void){
    int number=1;

    while(number < 51){
        if(number % 3 == 0 && number % 5 == 0){
            printf("FizzBuzz\n");
        }else if(number % 5 == 0){
            printf("Buzz\n");
        }else if(number % 3 == 0){
            printf("Fizz\n");
        }else{
            printf("%d\n",number);
        }
            number++;
        }
    return 0;
}